WIP Inclusive Events Schedule and Agenda Subsection (#3337)
the-turing-way/the-turing-way: book/website/collaboration/inclusive-events/inclusive-events-schedules.md

# Inclusive Event Schedules and Agenda

> When developing a schedule or agenda for your event, it is important to reflect on how diferent aspects of an event's schedule will impact how inclusive and accessible your event will be.

Reviewer: Typo: When developing a schedule or agenda for your event, it is important to reflect on how **different** aspects of an event's schedule will impact how inclusive and accessible your event will be.

Author: Hiya Giulia! 

I have a sneaky suspicion that this sentence has been revised or changed (as I can no longer find it) 🙈

As organizers, you should think critically about the timings and design of your event and and how to accommodate the diverse needs of your attendees. This can involve considerations ranging from planning sufficient breaks between sessions, to developing a plan for how to support the engagement and involvement of people with lived experience.

> This section provides guidance and reccomendations for how to plan an inclusive and and accessible event schedule (or agenda) and highlights what steps you can take to create a welcoming atmosphere where attendees with lived experience feel heard and valued, and are able to equitable engage in your event.

Reviewer: Typo: This section provides guidance and **recommendations** for how to plan

> It is crucial in your planning to prioritize inclusivity and accessibility in your event's scheduling and programming, and to recognize that the many of the traditional approaches to schedules and agendas may create barriers or be inaccessible to people with lived experience or disabilities. 

Reviewer: delete: and to recognize that **the** many of the traditional approaches

## Event Design: Timing and Event Duration

> When developing your event's program or agenda, you should consider the timing of your event's content. This includes identifying when your event will start and end, the duration of your event's content and how you will structure the event content to make it accessible, and finally the duration of the event itself.

Reviewer: Typo: This includes identifying when your event will start and end, the duration of your event's content and how you will structure the event**'s** content to make it accessible, and finally the duration of the event itself.
Comment: wondering if itemising might make it easier to read? but it's just a personal impression :)

If your event needs to start early in the morning, consider hosting a welcome session that is able to be missed if necessary. 

Additionally, consider planning your event around important holidays and cultural celebrations. By considering when your attendees will be required elsewhere, you will be more likely to find a time that suits everyone. 

### Session Duration
Your schedule should take into account breaks, meal times, and carefully plan the length of its sessions. 

If your event takes place in person, make sure to allow for sufficient time for people with lived experience and mobility difficulties to move between locations. For example, consider how long it would take to get to an accessible elevator and then travel between floors, or how far away accessible restrooms are. If your main room has only one exit and a large audience, it may take significantly longer to exit the room and cause attendees stress as they have to rush between sessions. 

Additionally, many people who experience cognitive or physical fatigue find it difficult to actively engage in events that last longer than 40 minutes without a meaningful break. 

> This timing can vary depending on individuals, however both ADHD advocates and people with a lived experience of cognitive impairements (such as stroke, traumatic brain injury, and brain fog) have advocated for 30-40 minute session blocks for activities that require sustained attention and engagement. 

Reviewer: Typo: with a lived experience of cognitive **impairments**

Try to plan for the most important parts of your event's content being shared within a blocks of time that are under 40 minutes.

> Because many events plan in 1 hour or 1.5 hour timing blocks, consider choosing shorter sessions or incorporating breaks within your session. This can be  as well as and allowing for 5-10 minutes for an introduction and session set up. This can help reduce the cognitive strain that many people with lived experience of cognitive impairements are affected by. 

Reviewer: This can be  as well as ~~and~~ allowing for 5-10 minutes for an introduction and session set up. This can help reduce the cognitive strain that many people with lived experience of cognitive **impairments** are affected by.

> **Example 1: 1 hour Focus Group Event**
    - 10 minute welcome, Code of Conduct, and Session introduction
    - 40 minute Discussion between participants
    - 10 minute break or 10 minute thank you and session close

> **Example 2: 1 hour Workshop event**
    - 5 minute introduction 
    - 20 minute session
    - 5 minute break
    - 20 minute session
    - 10 minute break or 10 minute thank you and session close

> It is best to get feedback and input from a representative with lived experience when planning your session timings. This is because there may be diferences in what is required for a specific community that would negatively impact a different set of individuals. 

Reviewer: "This is because there may be **differences** in what is required for a specific community that would negatively impact a different set of individuals."
This sentence sounds a bit unclear, but it might be just me! Maybe something like: "This is because certain requirements could be quite specific to the communities the event is addressing." ?

Author: I added in some clarifying sentences (and borrowed your phrasing as it was much clearer) :)

> For example, if your event invites specifically people with lived experience of ADHD (and no other cognitive impairments) for a creative workshop. You may want to consider offering longer sessions that are facilitated with self-directed breaks. For example, for a 1.5hr event you may consider a 10 minute introducion, followed by a 50 minute self directed session, a 15 minute break, and then a 15 minute conclusion or share out and session close. This is because many people with a lived experience of ADHD can find that forced interruption is very disruptive and prohibitive of meaningful engagement on tasks that are creative or require deep focus.

Reviewer: For example, for a 1.5hr event you may consider a 10 minute **introduction**

### Event Duration
In addition to session length, it is also important to consider how your event will be structured to support the involvement and engagement of people with lived experience. 

If you are organizing a multi-day conference, consider picking either a single day or a repeating timeslot that features any content specifically important for people with lived experience. For example, Day 1 of a 3-Day event might schedule the majority of sessions focused on public engagement and lived experience, followed by Day 2 which could focus on technical workshops and other content. Alternatively, sessions from 10 - 1pm on all 3 days could focus on lived experience content. 

This can support the engagement of people with lived experience who volunteer their time to engage in research, as well as people with lived experience of disabilities and medication conditions that can significantly impact their energy levels. 

## Event Content: Planning for Engagement

> When identifying the content of your event, you should spend time identifying what type of content and sessions will attendees with lived experience be interested in, what "engagement" means for your specific event, and how will you ensure that attendees with lived experience are able to engage in your event.

Reviewer: When identifying the content of your event, you should spend time identifying what type of content and sessions ~~will~~ attendees with lived experience **will** be interested in, what "engagement" means for your specific event, and how ~~will~~ you **will** ensure that attendees with lived experience are able to engage in your event.

> By clearly defining what engagement looks like, and also how to engage, your event will be make more accessible to people with lived experience and also reduce potential barriers and anxieties for participants.

Reviewer: By clearly defining what engagement looks like, and also how to engage, your event will be ~~make~~ more accessible to people with lived experience and also reduce potential barriers and anxieties for participants.

This can involve considerations ranging from planning sufficient breaks between sessions to developing a plan for how to support the engagement and involvement of people with lived experience.

This section provides guidance and recommendations for how to plan an inclusive and accessible event schedule (or agenda) and highlights what steps you can take to create a welcoming atmosphere where attendees with lived experience are heard and valued, and are able to equitably engage in your event.

It is crucial in your planning to prioritize inclusivity and accessibility in your event's scheduling and programming, and to recognize that many of the traditional approaches to schedules and agendas may create barriers or be inaccessible for people with lived experience or disabilities. 

(cl-inclusive-events-schedule-timing)=
## Event Design: Timing and Event Duration
When developing your event's program or agenda, you should consider the timing of your event's content. 

> This includes identifying when your event will start and end, the duration of your event's content and how you will structure the event content to make it accessible, and finally the duration of the event itself.

Author: shortening it works. The order didn't matter it was more a "here is everything this section will discuss" sentence

Additionally, consider planning your event around important holidays and cultural celebrations, as well as considering how weekend events will impact different people's ability to attend. 
Events that occur on weekends or during working hours can negatively impact the ability of people with lived experience who have jobs or caregiving commitments to attend events. 
By considering when your attendees will be required elsewhere, you will be more likely to find a time that suits everyone. 

(cl-inclusive-events-session-duration)=
### Session Duration
Your schedule should take into account breaks and meal times, and carefully plan the length of its sessions. 

If your event takes place in person, make sure to allow for sufficient time for people with lived experience and mobility difficulties to move between locations. 
For example, consider how long it would take to get to an accessible elevator and then travel between floors, or how far away accessible restrooms are. 
If your main room has only one exit and a large audience, it may take significantly longer to exit the room and cause attendees stress as they have to rush between sessions. 

> Additionally, many people who experience cognitive or physical fatigue find it difficult to actively engage in events that last longer than 40 minutes without a meaningful break. 

Author: I _almost_ do. But it's very much one of those that the true source has been lost to time and instead it's something that people ask for and notice. 

* I found some OG stroke research that pointed to the participants needing breaks at the 38-44 minute mark, but it wasn't the core point of the research so the authors added it almost as an afterthought. It feels weak to point to it directly as the main source
* I wanted to skirt around the "attention and productivity" literature as that was a minefield 
* I found some other suggestions from advocacy sites and university disability services suggesting 50 minute lectures or 45 mins, 10 min break, 35min for lectures lasting 1.5hrs. 

Basically I nerd-sniped myself for a day and a half trying to find a definitive source and ended up trying to skirt around having to cite as I couldn't find the exact thing I wanted without having to get into a messy discussion.

> Additionally, many people who experience cognitive or physical fatigue find it difficult to actively engage in events that last longer than 40 minutes without a meaningful break. 

Reviewer 2: Ooof. Then skip this comment. If I see anything in the meantime that would be more helpful to cite I can link it, but I don't remember anything that was this specific about it. Generally people just recommend shorter sessions...

This timing can vary depending on individuals, however, both ADHD advocates and people with a lived experience of cognitive impairments (such as stroke, traumatic brain injury, and brain fog) have advocated for 30-40 minute session blocks for activities that require sustained attention and engagement. 

> This timing can be highly dependant on the type of content and the specific audience you are engaging.

Author: done!

As a rule of thumb, try to plan for the most important parts of your event's content to be shared within blocks of time that are under 40 minutes.

Because many events are planned in 1-hour or 1.5-hour timing blocks, consider choosing shorter sessions or incorporating breaks within your session. 
This can be done in addition to allocating 5-10 minutes for an introduction and session set-up and will help reduce the cognitive strain that many people with lived experience of cognitive impairments are affected by. 

> **Example 1: 1-hour Focus Group Event**
    - 10 minute welcome, Code of Conduct, and Session introduction
    - 40 minute Discussion between participants
    - 10 minute break or 10 minute thank you and session close

> > **Example 2: 1-hour Workshop event**

Author: fixed!

    - 5 minute introduction 
    - 20 minute session
    - 5 minute break
    - 20 minute session
    - 10 minute break or 10 minute thank you and session close

> As mentioned above, it is best to get feedback and input from a representative with lived experience when planning your session timings. 

Author: I don't think so (unless you've found it since and then I stand corrected 🙈)

Do you think there should be an explicit suggestion (i.e. a new section) about how to engage and what engagement can mean. 

Right now there are just sentences and examples littered throughout. 

What do you think?

This is because there may be differences in what is required for a specific community that would negatively impact a different set of individuals. 

> For example, if your event invites specifically people with lived experience of ADHD (and no other cognitive impairments) for a :art: creative workshop :art:, setting sessions at 40 minutes maximum length may be incredibly disruptive and undermine the goals and accessibility of your workshop. 

Author: Added an admonition box. 

I'm not sure how it will render so happy to review this a few times before it gets finalized.

This is because many people with a lived experience of ADHD can find that forced interruption is very disruptive and prohibitive of meaningful engagement on tasks that are highly creative or require deep focus.
Instead, you may want to consider offering longer sessions for this audience that can be facilitated with self-directed breaks. 
For example, for a 1.5-hr event you may consider a 10 minute introduction, followed by a 50 minute self-directed session, a 15 minute break, and then a 15 minute conclusion or share out and session close.
This example highlights the importance of working with your specific community to identify what formats will work best for your specific event.
A discussion event or workshop for the same audience with ADHD may need to follow a different format (such a 30 minute instruction followed by a 10 minute break) to better support the audience's preferences and requirements.

> Try to get feedback on plans for your event as early as possible and ensure that you work with a representative or group with lived experience when deciding on the format and duration of your specific event. 

Author: Fixed (I think)

For example, people with an experience of cognitive impairment, as well as autistic people, can find unstructured sessions daunting which will negatively impact their ability to fully engage. 

The guidance you provide will be different depending on your event. 
For example, a public lecture may ask attendees to submit questions for the speaker at the end of the lecture. 
For this example, indicate in your guidance how long there will be for questions, how to use any online software you may be using to collect audience questions, and also how many questions you are likely going to answer. 
This can help avoid any feeling attendees may have about being ignored or not heard if their question is not addressed. 

For other events that require active participation, such as a focus group or discussions, consider including guidance on the format discussions will have, and if there are any specific topics or prior material attendees should be familiar with in order to take part. 

> Try to identify if your event will facilitate discussion between attendees, or if the sessions are to be chaired and led by a facilitator. 

Author: Based on this comment (and your insight) I think we'll add a section to the chapter which covers "guide for facilitators and moderators" and will build out that idea bit more 🚀 

I'll leave this comment unresolved for now so that we can cross-reference that new section once it's drafted in here.

Communicating to attendees that participation is expected, and what the specific format of participation will be, can help alleviate worry or anxiety that attendees may have, and also set the session up for success. 

(cl-inclusive-events-engagement-pathways)=
### Establishing Pathways for Engagement
People with lived experience may have varying comfort levels with public events, in addition to concerns about how any disabilities have been supported (or not) in past events.
Consider providing explicit guidelines and options for engagement to ensure that everyone is able to contribute at their own pace and in a manner that suits their preferences and needs. 

For example, a discussion or contribution-based event may consider offering both spoken and written contributions. 
"Speaking" group(s) can be facilitated by a note taker, and the "Writing" group(s) can work together in collaborative documents. 

> You may also consider recording your event's content and making documents available online as part of supporting asynchronous 

Author: 100% thank you for catching this.

contributions from people in multiple time zones or who may experience difficulties in contributing during "live" events. 

Consider designing sessions that support multiple channels of engagement to be as inclusive of as many accessibility requirements and preferences as possible. 

> Because all events are limited by the resources available to plan and host the event, it is important to provide clear guidance on what engagement means for your event and to work with a representative with lived experience to identify what methods of engagement your event will need to support.

Reviewer 2: I feel the limited resources heads up might have to go up earlier as that is a general limitation of inclusion/accessibility of events?

Author: I added a bit more to the sentence above to help it flow better. 

I'm also happy to take out the first half of the sentence on line 111 - but I feel like it's important to acknowledge that perfect may not be possible based on available resources for this suggestion.

Ensure that you provide an outline of your guidance in both a written format before the event (preferably in plain text), and also include an explanation in your event introduction of what engagement is expected, as well as what pathways are available for contributions.

(cl-inclusive-events-defining-goals)=
### Sharing the Goals of your Event
Consider outlining and sharing a set of goals or planned outputs from your event with your audience when you begin a session. 
For example, a workshop may present learning objectives or an example of the planned finished product. 

By clearly articulating the event's objectives, you will be able to create a sense of transparency and purpose for the session. 
This clarity allows all attendees, including people with lived experience, to understand what to expect from the session and reflexively evaluate whether their participation aligns with those goals. 

Additionally, sharing an event's goals can foster a sense of trust among attendees by presenting how the perspectives and contributions of people with lived experience are valued by the research project team. 

> This can be especially important for "data-driven" research that otherwise places a large focus on algorithmic techniques. 

Reviewer 2: To me this sentence comes a bit out of the blue. It is true, but why add this particularly here?

Author: Added a bit more context.

Inclusive research requires not only active participation from people with lived experience and diverse backgrounds, but also actively ensuring that their opinions and perspectives are heard and able to influence the research directly. 
By orientating your attendees around a goal, you can provide a shared understanding that will support meaningful and inclusive engagement in your event.

(cl-inclusive-events-inclusive-content)=
### Planning for Inclusion
As part of planning the program, consider how you will support and facilitate relationship-building and the active inclusion of people with lived experience in your event's sessions.

Research events can often be focused on actively engaging researchers who have a clear pathway to engagement and involvement in an event, for example explaining their research or asking questions of colleagues to better understand what research methods led to a new discovery. 

> If your event plans to involve people with lived experience, you should consider designing your event's sessions to allow for experts through experience to engage in the material equitably alongside any attendees who are experts by profession, instead of being segregated out for being "non-researchers". 

Reviewer 2: Oof yes. 
Maybe also add something on how to ensure that both these perspectives are considered as equal and the researcher perspective should not be leading? I feel like this should be stressed more (based on some experiences from the same session last week...)

You can read more about ADIDS on the [MozFest website](https://www.mozillafestival.org/en/facilitator-guide/designing-a-successful-session/), or by working through the framework for your own event.
```
(cl-inclusive-events-schedule-rapport)=
### Building Rapport with Event Design
The active involvement of people with lived experience is incredibly important because it enables the design and delivery of research and outputs that are informed by lived experiences, and better reflects the needs of the communities that stand to benefit the most from research interventions.

You can encourage inclusive participation from people with lived experience by making use of inclusive and "unconventional" approaches in your event. 
For example, consider brokering conversation between attendees in a "World Cafe". 

World Cafes bring rotating tables (virtual or physical) of people with differing expertise together to engage in evolving rounds of dialogue on a specific topic. 
The smaller grouping allows for more conversation between people and can be less daunting for people with lived experience to engage in. 

> You can read more about the World Cafe method on [The World Café Community Foundation's blog](https://theworldcafe.com/key-concepts-resources/world-cafe-method/). 

Reviewer 2: Thank you for making me read more! ;)

(cl-inclusive-events-moderators-facilitators)=
### The role of Moderators and Facilitators
Facilitators and session moderators can help to foster a safe and supportive environment for people with lived experience to share their perspectives and insights. 

> Their role at an inclusive event is to provide safe and supported opportunities for active participation from attendees, act as a catalyst for conversation, and to help align discussion and any questions, with the goals of the session and event. 

Reviewer 2: ```suggestion
Their role at an inclusive event is to provide safe and supported opportunities for active participation from attendees, act as a catalyst for conversation, and to help align discussion and any questions with the goals of the session and event. 
```

If you are able to involve facilitators or moderators at your event, consider how they can support the engagement of - and contributions from - people with lived experience. 
This is especially important for events that focus on discussion or require active participation (such as the World Cafes we discussed in the previous section). 
Many people with lived experience may not have a lot of experience, or had the opportunity to develop skills in public speaking or rhetoric and can therefore be at a greater risk of getting left out of conversation. 

> Just inviting people with lived experience to your event is not sufficient if they are not supported and actively welcomed to engage in your event's content. 

Reviewer 2: I feel something should be added that the burden to participate should not be on those with lived experience, but should be carried by the facilitators which should also read up on neurodiversity and barriers to participation. This links in with the earlier comment I had on what facilitators can really do to help here - it is beyond facilitation of just neuro-typicals.. Sorry just rambling here and jotting this down for myself as well so that I can hopefully add this later.

This can help build rapport among event attendees, and orientate everyone towards a shared goal before the formal parts of your session begin.

By establishing an active partnership between people with lived experience, researchers, and other experts by profession in your event's sessions, you will be well-positioned to support an equitable space for all attendees to meaningfully engage with your event.

(cl-inclusive-events-inclusive-program-planning)=
## Creating an Inclusive Program
If you are able, consider including a person with lived experience in the planning of an event's program and agenda. 

Involving people with lived experience in the planning and delivery of events ensures the voice of people with lived experience is built into the dialogue throughout your event. 

Active involvement can also help organizers develop a greater understanding of what is important from the perspective of people with lived experience and ensure that the proposed structure and content of the event is shaped in a way that will meet the needs of the potential attendees.

> Their involvement can offer insight and perspective into the specific challenges and needs of individuals with disabilities and any specific lived experience your event is focused on. If you are including a person, or group, with lived experience in the planning, ensure that you are appropriately compensating and supporting them for their expertise.

Reviewer 2: ```suggestion
Their involvement can offer insight and perspective into the specific challenges and needs of individuals with disabilities and any specific lived experience your event is focused on. 
If you are including a person, or group, with lived experience in the planning, ensure that you are appropriately compensating and supporting them for their expertise.
```

> If you are unable to include a representative on your planning team, try to prioritize the engagement of, and feedback from, the specific communities that will be in attendance. People with lived experience may offer different ideas about how to present the materials, and their insights will be invaluable in developing an inclusive event program that is genuinely accessible, respectful, and responsive to the diverse needs of all participants.

Reviewer 2: ```suggestion
If you are unable to include a representative on your planning team, try to prioritize the engagement of, and feedback from, the specific communities that will be in attendance. 
People with lived experience may offer different ideas about how to present the materials, and their insights will be invaluable in developing an inclusive event program that is genuinely accessible, respectful, and responsive to the diverse needs of all participants.
```
I feel this should also say something like that even if you manage to have a representative on the planning team, they may not be familiar with all the requirements and needs which may be very diverse. So asking for feedback from other participants remains key.

> When developing your event's program or agenda, ensure that you have established a pathway or mechanism to get feedback and input from people with lived experience. The two options discussed above are ideal, as people with lived experience will be able to provide insights and perspectives on the specific challenges and needs of your attendees. This perspective can be helpful when identifying what sessions or content should be included in the program, and will support you in creating a more inclusive and relevant program for your audience. 

Reviewer 2: ```suggestion
When developing your event's program or agenda, ensure that you have established a pathway or mechanism to get feedback and input from people with lived experience. 
To receive this feedback and input you can include people with lived experience in the planning committee of your event, and you can ask potential attendees for feedback on the planning and materials of your event. 
```
I would add this as the first sentence of this section, so you introduce a bit better what this section is about. My suggested sentence probably needs an update...

Author: Made some changes :)

This is important because even in research projects that involve participatory methods, not all researchers have the opportunity to directly engage with the people with lived experience involved in the project. 
Having a speaker or session lead present from a place of lived experience can give all attendees the opportunity to learn and engage with the process of participatory research as a whole, and ensures that researchers, research staff, and other professionals will develop a more comprehensive understanding of the opportunities and challenges faced by people with lived experience. 

This may also have the added benefit of improving the research conducted by attendees, or influence project design in the future. 

Many research conferences and events provide opportunities for discussion and the learning of different research skills and practices. 
Lived experience speakers can complement and enrich research insights, and offer a more holistic and authentic understanding of different contexts and experiences. 

> Including and highlighting speakers or session leads with lived experience and offering a diverse range of voices both broadens the scope of representation, and helps to prevent assumptions and stereotypes about people with lived experience which can further help to empower other attendees with lived experience to become more actively involved in research.

Author: I reworked it a bit. 

I'm worried that the sections are now a bit too long nut tell me what you think.

 
(cl-inclusive-events-event-descriptions)=
### Providing Event Descriptions
Consider including a description of your intended audience and any expertise requirements in your session information. 

> Ensure that these session descriptions, the program, and any organisational information are written using Plain Language. 

Author: Changed it. 

I think the Lay summaries section may need to be expanded upon a bit.

> **Facilitators:** Alan Turing, Ada Lovelace, Charles Babbage, Katherine Johnson
> **Audience:** Technical leads, statisticians, and data-intensive researchers across the AIM consortia
> **Time:** 14.00 - 15.30. Comfort break from 14:45 - 15.00. 
> **Location:** The Great Hall, Hogwarts.

```{note}
You can find other inspiration and examples of inclusive session descriptions on the [Mozilla Festival (MozFest) website](https://schedule.mozillafestival.org/schedule).]
```

(cl-inclusive-events-parallel-sessions)=
### Using Parallel Sessions
Conferences and other large events can often bring together a wide range of attendees with an equally wide range of experiences and expertise. 
For these events, it may not be ideal to curate all content and sessions suitable for a general audience. 
However, in this case, it is very important that organizers clearly communicate about the content of sessions, and that sessions are labelled appropriately on all promotional material. 

> Consider if you are able to run "parallel" tracks or sessions and dedicate one of these tracks to sessions that would be of interest to people with lived experience. 

Author: Hi @EstherPlomp 

I'm going to merge this PR but am adding this comment to a new task list which we can address as part of the edit/re-write in the new year.

Ensure that you provide an outline of your guidance in both a written format before the event (preferably in plain text), and also include an explanation in your event introduction of what engagement is expected, as well as what pathways are available for contributions.

(cl-inclusive-events-defining-goals)=
### Sharing the Goals of your Event
Consider outlining and sharing a set of goals or planned outputs from your event with your audience when you begin a session. 
For example, a workshop may present learning objectives or an example of the planned finished product. 

By clearly articulating the event's objectives, you will be able to create a sense of transparency and purpose for the session. 
This clarity allows all attendees, including people with lived experience, to understand what to expect from the session and reflexively evaluate whether their participation aligns with those goals. 

> 

Reviewer 2: ```suggestion
Additionally, consider if your goals are measurable so that you can more easily conclude after the event whether the goals were reached. 
```
Or something like that - from the OSFNL crew

This may also have the added benefit of improving the research conducted by attendees, or influence project design in the future. 

By including and highlighting speakers or session leads with lived experience and offering a diverse range of voices both broadens the scope of representation, and can prevent assumptions and stereotypes about people with lived experience.
This can then act to further empower other attendees with lived experience to become more actively involved in your event and research as a whole.

(cl-inclusive-events-event-descriptions)=
### Providing Event Descriptions
Consider including a description of your intended audience and any expertise requirements in your session information. 
Ensure that these session descriptions, the program, and any organisational information are written using [{Plain Language}`def<Plain Language>`] that is able to be understood by your audience. 

Include session descriptions in the program of events. This description should include:
- a short and clear title of the session or event
- the name of any hosts, facilitators, or moderators

> - a short abstract about what the session is 

Reviewer 2: and what the goal of the session is? (the OSFNL crew focused more on this)

Author: done ✨

for how to support the engagement and involvement of people with lived experience.

This section provides guidance and recommendations for how to plan an inclusive and accessible event 
schedule (or agenda) and highlights what steps you can take to create a welcoming atmosphere where attendees 
with lived experience feel heard and valued, and are able to equitably engage in your event.

It is crucial in your planning to prioritize inclusivity and accessibility in your event's scheduling and 
programming, and to recognize that many of the traditional approaches to schedules and agendas may create 
barriers or be inaccessible for people with lived experience or disabilities. 

(cl-inclusive-events-schedule-timing)=
## Event Design: Timing and Event Duration
When developing your event's program or agenda, you should consider the timing of your event's content. 
This includes identifying when your event will start and end, the duration of your event's content and 
how you will structure the event content to make it accessible, and finally the duration of the event itself.

(cl-inclusive-events-start-times)=

> ### Start and End Times

Author: It was stated elsewhere, but I'm trying as much as possible to assume that people may not read every section so restate things in parts. I'll add that comment in again here into this section.

    - 40 minute Discussion between participants
    - 10 minute break or 10 minute thank you and session close

> **Example 2: 1-hour Workshop event**
    - 5 minute introduction 
    - 20 minute session
    - 5 minute break
    - 20 minute session
    - 10 minute break or 10 minute thank you and session close

It is best to get feedback and input from a representative with lived experience when planning your session timings. 
This is because there may be differences in what is required for a specific community that would negatively 
impact a different set of individuals. 

For example, if your event invites specifically people with lived experience of ADHD 
(and no other cognitive impairments) for a creative workshop. 
You may want to consider offering longer sessions that are facilitated with self-directed breaks. 
For example, for a 1.5-hr event you may consider a 10 minute introduction, followed by a 50 minute self-directed session, 
a 15 minute break, and then a 15 minute conclusion or share out and session close. 
This is because many people with a lived experience of ADHD can find that forced interruption is very disruptive and 

> prohibitive of meaningful engagement on tasks that are creative or require deep focus.

Author: I added a few sentence throughout this section to try add some clarity. 

Let me know if what I added was 

- [ ] too much (aka, over stating the same thing) 
- [ ] helpful for making it clearer

(cl-inclusive-events-event-duration)=
### Event Duration
In addition to session length, it is also important to consider how your event will be structured to support the 
involvement and engagement of people with lived experience. 

If you are organizing a multi-day conference, consider picking either a single day or a repeating timeslot 
that features any content specifically important for people with lived experience. 
For example, Day 1 of a 3-Day event might schedule the majority of sessions focused on public engagement and lived experience, 
followed by Day 2 which could focus on technical workshops and other content. 

> Alternatively, sessions from 10 a.m. - 1 p.m. on all 3 days could focus on lived experience content. 

Reviewer 3: For this I'm not sure about how good that would work in practice. At least if that's the only content for PPI members it means they have to travel/get accommodations, carers etc. every day. But I guess that goes back to "ask the people with lived experience what works for them" instead of assuming :)

Author: Added in a sentence as I think it always bears repeating.

The smaller grouping allows for more conversation between people and can be less daunting for people 
with lived experience to engage in. 
You can read more about the World Cafe method on [The World Café Community Foundation's blog](https://theworldcafe.com/key-concepts-resources/world-cafe-method/). 

(cl-inclusive-events-moderators-facilitators)=
### The role of Moderators and Facilitators
Facilitators and session moderators can help to foster a safe and supportive environment for 
people with lived experience to share their perspectives and insights. 
Their role at an inclusive event is to provide safe and supported opportunities for active 
participation from attendees, act as a catalyst for conversation, and to help align discussion 
and any questions, with the goals of the session and event. 

If you are able to involve facilitators or moderators at your event, consider how they can support 
engagement of and contributions from people with lived experience. 
This is especially important for events that focus on discussion or require active participation 
(such as the World Cafes we discussed in the previous section). 

> Many people with lived experience may not have recieved formal training in public speaking or 

Reviewer 3: ```suggestion
Many people with lived experience may not have received formal training in public speaking or 
```

rhetoric, and can be at a greater risk of being left out of conversation. 

Just inviting people with lived experience to your event is not sufficient if they are not 
supported and actively welcomed to engage in your event's content. 

Although it is becoming more common for people with lived experience to become self-advocates and receive 
training in public speaking, it is important for researchers and event organizers to consider - and make use of - 

> moderators and session facilitators in order to uphold inclusive and equitable engagement practices. 

Author: I'll see if I can rework this section a bit ⭐

(cl-inclusive-events-icebreakers)=
### Using Icebreakers and Teambuilding
Additionally, you may consider using a collaborative activity to open your session 
such as an icebreaker or other team-building task. 
This can help build rapport among event attendees, and orientate everyone towards a shared goal 
before the formal parts of your session begin.

By establishing an active partnership between people with lived experience, researchers, 
and other experts by profession in your event's sessions, you will be well-positioned to support 
an equitable space for all attendees to meaningfully engage with your event.

## Including People with Lived Experience in Session Planning / Creating an Inclusive Program

> [See Draft PR Discussion - section content WIP]

Reviewer 3: I think option A works 👍

Author: Added
